### DISCUSSION TITLE
I want to remove "Registering datasource 'default' with db provider: 'Quartz.Impl.AdoJobStore.Common.DbProvider'" Log from Log File. Is there any option to avoid this from printing in Main Log File

### DISCUSSION THREAD

**Discussion by SuvinayPisal** [quartznet/quartznet#1317] [Q&A]
I want to remove this from Printing into log file..
is it possible?

Registering datasource 'default' with db provider: 'Qu****************************Provider'

**Comment by lahma** [comment]
Probably not going to be easy to remove just one line and keep all the others seen here. Why do you want to do so?

**Reply by SuvinayPisal** [reply]
I want neat and clean Log so i dont want to save stuff that interrupt serial data..

I just want to save data in a {}


For example:
//Log data
{Some data:233;  some data :333}
{Some data:233;  some data :333}
{Some data:233;  some data :333}
{Some data:233;  some data :333}


//which i dont want to log
Registering datasource 'default' with db provider: 'Qu****************************Provider'
//End which i dont want to log

//Log data
{Some data:233;  some data :333}
{Some data:233;  some data :333}
{Some data:233;  some data :333}
{Some data:233;  some data :333}

**Reply by lahma** [reply]
So why don't you just set Quartz category logging level to Warning or Error in your logging configuration?
